10-stretch-slim/Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN bash /tmp/consul_template_install.sh && rm /tmp/consul_template_install.sh
 # We use the following variables both in here and in downstream Dockerfiles
 ENV SERVICE_ROOT /service
 ENV SERVICE_USER service
+ENV NODE_MODULES_PATH /node_modules
```

Comment:
I think you want `ENV PATH $PATH:/node_modules/.bin` so it appends this to the path ... what are you expecting to use `NODE_MODULES_PATH` for?